src/linear_samplers/vec_col_one_rand_cyclic.jl

```diff
@@ -0,0 +1,50 @@
+# This file is pat of RLinearAlgebra.jl
```

Comment:
I made a mistake here and wrote "pat" instead of "part"

Reply:
@vp314 noted and modified